setup.py

```diff
@@ -19,6 +19,7 @@
 from setuptools import setup
 
 if sys.version_info[:2] < (2, 7):
+    # TODO: Replace with `python_requires` in `setup`
```

Comment:
Here we go:
`python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*',`

Reply:
hehe. wonderful.